packages/gatsby-theme-newrelic/README.md

```diff
@@ -236,6 +237,23 @@ documentation.](https://www.gatsbyjs.org/packages/gatsby-plugin-robots-txt/)
 Optional configuration for related resources used in the right rail. Currently
 only `Mdx` nodes are supported.
 
+The related resources component is controlled by specific Frontmatter slugs that
+are defined on a page. By setting the Frontmatter for `resources` and `tags` you can control what is populated in this component.
+
+If no resources or tags are available in the page Frontmatter the component will fallback to using the page title as the search query term and related resources will be auto generated.
+
+The order of priority for populating content in this component is driven by:
+
+1. Any resources defined in the page Frontmatter.
+2. Any tags defined in the the page Frontmatter will send a search query term and return results from Swiftype.
+   2b. Any Swifttype results that match the page title.
+
+Resource site tags control the labels that appears below each resource URL. These can be found in the `Resources.js` in the specific site repository.
```

Comment:
I'd remove "site tags" as a description here since it might be a bit confusing. We are now calling these `labels`. You can find the default label mapping here: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/utils/defaultOptions.js#L10-L23

That being said, I may advocate for removing this paragraph since talked about in the `labels` config in the `Options` section below. If the link to the defaults is helpful, perhaps we update that little section?

Reply:
will fix